.editorconfig

```
@@ -0,0 +1,32 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+tab_width = 4
+
+[*.gradle]
+indent_style = tab
+
+[*.java]
+indent_style = tab
+max_line_length = off
```

```suggestion
max_line_length = 2147483647
```
based on my experience, intellij seems to bug a bit with it at off, but doesn't when using the int32 limit

---

It seems fine, can you clarify?

---

Basically it works most of the time, but i have seen IJ mess up and still hard wrap lines despite it being set to off
It happened consistently even after reloading IJ, setting it manually in the settings instead of editorconfig
I'm gonna try to find an exact snippet that caused the issue
(pain applying the editorconfig retroactively changed 575 files)